Logstash/logstash-input-azureblob/README.md

```diff
@@ -56,8 +56,37 @@ When set to `start_over`, it assumes none of the blob is consumed and it will re
 
 Offsets will be picked up from registry file whenever it exists.
 
+__*file_head_bytes*__
+
+Specifies the header of the file in bytes that does not repeat over records. Usually, these are json opening tags. The default value is `0`.
+
+__*file_tail_bytes*__
+  
+Specifies the tail of the file that does not repeat over records. Usually, these are json closing tags. The defaul tvalue is `0`.
+
+### Advanced tweaking parameters
+
+Keep these parameters default to use under normal situration. Tweak these parameters when dealing with large scale azure blobs and logs.
+
+__*blob_list_page_size*__
+
+Specifies the page-size for returned blob items. Too big number will hit heap overflow; Too small number will leads to too many requests. The default of `100` is good for heap size of 1G.
+
+__*break_json_down_policy*__
+
+Only works when the codec is set to `json`. Sets the policy to break the json object in the array into small events. Break json into small sections will not be as efficient as keep it as a whole, but will reduce the usage of the memory. Possible options: `do_not_break`, `with_head_tail`, `without_head_tail`. 
```

Comment:
It would be helpful to give an example of how it breaks them down. Like how it defines the boundaries between different events.

Reply:
Good idea. Worth a wiki page for this.